src/helpers/github.members.ts

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
+
+/**
+ * Github helper class, use for managing API called in github
+ * @export default
+ * @class GithubHelper
+ */
+export default class GithubMembers {
+  /**
+ * Get all members of Public members of Rwanda Open Source
+ * return -1 when some error occured
+ * @static
+ * @returns {array} github user accounts
+ * @memberof GithubMembers
+ */
+  static async viewAllPublicMembers(): Promise<number> {
+    try {
+      const URL = 'https://api.github.com/orgs/rwandaopensource/members';
```

Comment:
Good Job @Ntare22, However, I think this URL should come from `.env` file to better secure our app.

Reply:
Thanks... @william000000 .... I'll correct on the next PR.. this one was merged

Reply:
Makes sense